scripts/setWorkingdirMode.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )"/.. && pwd )"
+
+if [ -z "$EXPO_DOWNLOAD_DIR" ]; then
+  DOWNLOAD_DIR="$HOME/Downloads"
+else
+  DOWNLOAD_DIR=$EXPO_DOWNLOAD_DIR
+fi
+mkdir -p $DOWNLOAD_DIR
+
+if [ ! -f "$ROOT_DIR/.secrets.local-queue" ]; then 
+  echo "You need to run \"yarn secrets:init-private [SQS_NAME]\" or create .secrets.local-queue file yourself with variables AWS_SQS_ANDROID_QUEUE_URL, AWS_SQS_IOS_QUEUE_URL and AWS_SQS_OUT_QUEUE_URL."
+  exit 1
+fi
+
+setup_local() {
+  ensure_link_exists() {
+    if [ ! -L "$2" ]; then
+      ln -s $1 $2
+      echo "$2 link created"
+    fi
+  }
+  WORKINGDIR="$ROOT_DIR/workingdir/local"
+  
+  if [[ -d $WORKINGDIR ]]; then
+    echo "Lokal workingdir already prepared"
+    return
+  fi
+
+  if [[ -z "$EXPO_UNIVERSE_DIR" ]]; then
+    echo "EXPO_UNIVERSE_DIR needs to be set"
+    exit -1
+  fi
+
+  cd $EXPO_UNIVERSE_DIR/exponent && yarn
+  cd $EXPO_UNIVERSE_DIR/exponent/tools-public && yarn && ./generate-dynamic-macros-android.sh
+
+  mkdir -p $WORKINGDIR
+  echo "$WORKINGDIR directory created"
+
+  LINKS=("template-files" "package.json" "android" "expokit-npm-package" "packages" "client-builds" "cpp" "exponent-view-template" "ios" "shellAppBase-builds" "shellAppWorkspaces")
+
+  for LINKNAME in ${LINKS[@]}; do
+    ensure_link_exists "$EXPO_UNIVERSE_DIR/exponent/$LINKNAME" "$WORKINGDIR/$LINKNAME"
+  done
+}
+
+setup_remote() {
+  if [[ -d $ROOT_DIR/workingdir/android && -d $ROOT_DIR/workingdir/ios ]]; then
+    echo "Shellapps already fetched"
+  else
+    echo "Fetching shellapps"
+    $ROOT_DIR/scripts/android/fetchRemoteAndroidTarball.sh
+    $ROOT_DIR/scripts/ios/fetchRemoteIosTarball.sh
```

Comment:
Since this script is used when creating a Turtle agent tarball we can't just fetch tarballs for both iOS and Android.